File: Assets/Scripts/GamePlayRoom1/Door.cs
using UnityEngine;
using System.Collections;

public class Door : MonoBehaviour {
	public GameObject winPopup;
	GameObject thedoor;
	// Use this for initialization
	void Start () {
		thedoor= GameObject.Find("door");
	}

	void OnTriggerEnter(Collider coll) {

		if (coll.gameObject.tag == "Player") {
			
			thedoor.GetComponent<Animation>().Play("open");
		winPopup.SetActive (true);

		}
	}
}

File: Assets/SciFi_Door/Script/door.js
#pragma strict


function OnTriggerEnter (obj : Collider) {
	var thedoor = gameObject.FindWithTag("SF_Door");
	thedoor.GetComponent.<Animation>().Play("openL");
}

function OnTriggerExit (obj : Collider) {
	var thedoor = gameObject.FindWithTag("SF_Door");
		thedoor.GetComponent.<Animation>().Play("closeL");
	
}